### PR TITLE
Feature/extra_installs

### PR DESCRIPTION
**Pull request by Cybsloth**
## Summary

Separate installation of dependencies has become available.

Installation examples (after adding package to pypi):

```
pip install lightautoml[nlp]  # nlp part
pip install lightautoml[cv,nlp]  # cv and nlp parts
pip install lightautoml[all] # all 
```

If you try to import modules with missing dependencies, a warning will be displayed.
When trying to import presets that do not have the missing dependencies, an exception will be thrown.

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -43,29 +44,30 @@ networkx = "*"
...
+# CV
+albumentations = {version = ">=0.4.6", optional = true}
+efficientnet-pytorch = {version = "*", optional = true}
+opencv-python = {version = "*", optional = true}
+PyWavelets = {version = "*", optional = true}
+scikit-image = {version = "*", optional = true}
+torchvision = {version = "*", optional = true}
 
-importlib-metadata = {version = "^1.0", python = "<3.8"}
+# Report (pdf)
```
[the remark this answers is not in the document]
Comment (Cybsloth) [reply pyproject.toml:69]: fix

File: README.md [hunk 2]
```diff
@@ -16,27 +16,155 @@ Currently we work with datasets, where **each row is an object with its specific
...
 pip install -U lightautoml
+
+# Available partial installation
+# Use extra dependecies = ['nlp', 'cv', 'pdf']
+# Or may use 'all' for installation full functionality, example:
+
+pip install -U lightautoml[nlp]
+
 ```
 ### Installation from sources with virtual environment creation
```
[the remark this answers is not in the document]
Comment (Cybsloth) [reply README.md:198]: Updated

File: .flake8 [hunk 3]
```diff
@@ -1,5 +1,5 @@
 [flake8]
-max-line-length = 88
+max-line-length = 133
```
[the remark this answers is not in the document]
Comment (Cybsloth) [reply .flake8]: set to 120

**Comment by dev-rinchin** [comment]
LGTM, just resolve conflicts and merge.

**Pull request merged**